runtime/centrifuge/src/migrations.rs

```diff
@@ -10,11 +10,26 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
 
+use cfg_primitives::AccountId;
+use frame_support::parameter_types;
+use sp_std::{vec, vec::Vec};
+
 use crate::{OraclePriceCollection, OraclePriceFeed};
 
 // Number of identities on Centrifuge Chain on 29.05.2024 was 34
 const IDENTITY_MIGRATION_KEY_LIMIT: u64 = 1000;
 
+parameter_types! {
+	// Address used by Anemoy to withdraw in AssetHub
+	// 4dTeMxuPJCK7zQGhFcgCivSJqBs9Wo2SuMSQeYCCuVJ9xrE2 --> 5Fc9NzKzJZwZvgjQBmSKtvZmJ5oP6B49DFC5dXZhTETjrSzo
+	pub AccountMap: Vec<(AccountId, AccountId)> = vec![
+		(
+			AccountId::new(hex_literal::hex!("5dbb2cec05b6bda775f7945827b887b0e7b5245eae8b4ef266c60820c9377185")),
```

Comment:
This account is `4dTeMxuPJCK7zQGhFcgCivSJqBs9Wo2SuMSQeYCCuVJ9xrE2` in hex, right?

Reply:
Yes
```
❯ subkey inspect 4dTeMxuPJCK7zQGhFcgCivSJqBs9Wo2SuMSQeYCCuVJ9xrE2
Public Key URI `4dTeMxuPJCK7zQGhFcgCivSJqBs9Wo2SuMSQeYCCuVJ9xrE2` is account:
  Network ID/Version: centrifuge
  Public key (hex):   0x5dbb2cec05b6bda775f7945827b887b0e7b5245eae8b4ef266c60820c9377185
  Account ID:         0x5dbb2cec05b6bda775f7945827b887b0e7b5245eae8b4ef266c60820c9377185
  Public key (SS58):  4dTeMxuPJCK7zQGhFcgCivSJqBs9Wo2SuMSQeYCCuVJ9xrE2
```